.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This comes from VSCode config files. They are not needed in the commit as it "forces" an editor on the developers. Also they can change from one machine to another. I recommend you use a [.gitignore_global](https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) to automatically omit the .vscode folder when doing a commit. [Git ignore rules](https://www.atlassian.com/git/tutorials/saving-changes/gitignore).